packages/graphql-yoga/src/plugins/request-parser/post-multipart.ts

```diff
@@ -53,7 +53,7 @@ export async function parsePOSTMultipartRequest(request: Request): Promise<Graph
 
     try {
       map = JSON.parse(mapStr);
-    } catch (err) {
+    } catch {
```

Comment:
Feels like errors are being swallowed here?

Reply:
It was already swallowed, it is removed to fix linting errors;
https://github.com/dotansimha/graphql-yoga/pull/3532/files/9b8aaff82adde406e52e146e44d6fab71c961c2d#diff-945452c8fc94d4993c7c99f5ff592455352a72956bde64fc8a3db5336dadb949L57